### DISCUSSION TITLE
Can I use stella_vslam_ros support panoramic camera?

### DISCUSSION THREAD

**Discussion by Austin1207** [OpenVSLAM-Community/openvslam_ros#95] [Q&A] [answered]
I know that stella_vslam support panoramic camera, but it seems not to support in stella_vslam_ros, am I right?

**Comment by ymd-stella** [comment, marked as answer]
No, it is supported.

**Reply by Austin1207** [reply]
Thanks for your reply. I want to use theta z1 to run slam on ros. I have used the driver you supply on [theta_driver], and I get the 360 image with rostopic. My question is that whick mode should I choose in theta_vslam_ros,? Because I only see [monocular] [stereo] [RGBD] setup, thank you.

**Reply by ymd-stella** [reply]
monocular
